usaspending_api/common/etl/spark.py

```diff
@@ -292,7 +294,8 @@ def load_delta_table(
             If left False (the default), the DataFrame data will be appended to existing data.
     Returns: None
     """
-    logger = get_jvm_logger(spark)
+    logging.basicConfig(level=logging.INFO)
+    logger = logging.getLogger(__name__)
```

Comment:
This applies to most of the changes on the PR.

Since you are using the `__name__` for the logger (which is what we want) I would move this to the top of the file. Using `query_with_filters.py` as an example (https://github.com/fedspendingtransparency/usaspending-api/blob/255a032f8b80117d8dc6cf22852ba6f7126a0ce5/usaspending_api/common/query_with_filters.py#L30).